theano/tensor/nnet/abstract_conv.py

```diff
@@ -308,6 +308,17 @@ def perform(self, node, inp, out_):
             'Did you exclude both "conv_dnn" and "conv_gemm" from '
             'the optimizer? Is cudnn available and does the GPU support it?')
 
+    def R_op(self, inputs, eval_points):
```

Comment:
This would also need tests. rop tests are in theano/tests/test_rop.py grep for conv, you can probably copy/paste/modify (or better, generalize it to test both)